pyomo/core/expr/visitor.py

```diff
@@ -1392,108 +1410,83 @@ def __init__(self, include_fixed=False, named_expression_cache=None):
         """
         super().__init__()
         self._include_fixed = include_fixed
-        if named_expression_cache is None:
-            # This cache will map named expression ids to the
-            # tuple: ([variables], {variable ids})
-            named_expression_cache = {}
-        self._named_expression_cache = named_expression_cache
-        # Stack of active named expressions. This holds the id of
-        # expressions we are currently in.
-        self._active_named_expressions = []
+        self._cache = named_expression_cache
+        # Stack of named expressions. This holds the tuple
+        #     (eid, _seen, _exprs)
+        # where eid is the id() of the subexpression we are currently
+        # processing, and _seen and _exprs are from the parent context..
```

Comment:
```suggestion
        # processing, and _seen and _exprs are from the parent context.
```
NFC: Removing extra period